stderr of test 'XML_attribute_generation.SF-2038906-unpack_doc` in directory 'tests/BugTracker` itself:


# 13:12:44 >  
# 13:12:44 >  /usr/bin/python XML_attribute_generation.SF-2038906-unpack_doc.py XML_attribute_generation.SF-2038906-unpack_doc 
# 13:12:44 >  


# 13:12:44 >  
# 13:12:44 >  Done.
# 13:12:44 >  

